src/components/Onboarding/Screens/Apps/TokensSettingsNative.js

```diff
@@ -0,0 +1,450 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  EthIdenticon,
+  Field,
+  GU,
+  Help,
+  IconPlus,
+  IconTrash,
+  Info,
+  isAddress,
+  RADIUS,
+  TextInput,
+  useTheme,
+} from '@1hive/1hive-ui'
+import Navigation from '../../Navigation'
+import { useOnboardingState } from '../../../../providers/Onboarding'
+import Header from '../../kit/Header'
+
+function useFieldsLayout() {
+  return `
+    display: grid;
+    grid-template-columns: auto ${14.5 * GU}px;
+    grid-column-gap: ${1.5 * GU}px;
+  `
+}
+
+function validateDuplicateAddresses(members) {
+  const validAddresses = members
+    .map(([address]) => address.toLowerCase())
+    .filter(address => isAddress(address))
+
+  return validAddresses.length === new Set(validAddresses).size
+}
+
+function validationError(tokenName, tokenSymbol, members) {
+  if (members.some(([address]) => !isAddress(address))) {
+    return 'Every address you provide should be a valid address.'
+  }
+  if (members.some(([account, stake]) => !(stake > 0))) {
+    return 'Every account has to have a positive balance.'
+  }
+  if (!validateDuplicateAddresses(members)) {
+    return 'One of your members is using the same address than another member. Please ensure every member address is unique.'
+  }
+  if (!tokenName.trim()) {
+    return 'Please add a token name.'
+  }
+  if (!tokenSymbol) {
+    return 'Please add a token symbol.'
+  }
+  return null
+}
+
+function TokensSettingsNative() {
+  const theme = useTheme()
+  const fieldsLayout = useFieldsLayout()
+
+  const { onBack, onNext, steps, step, onConfigChange } = useOnboardingState()
+
+  const [formError, setFormError] = useState()
+
+  const [tokenName, setTokenName] = useState('')
+  const [tokenSymbol, setTokenSymbol] = useState('')
+  const [commonPool, setCommonPool] = useState(0)
+  const [accountStake] = useState(0)
+
+  const [members, setMembers] = useState([['', accountStake]])
```

Comment:
Same as in Token settings BYOT, we should use the config values.

Also, i don't think the `accountStake` variable is being updated si i think we can remove it and use directly 0 ?

Note that for `holders` we need to update the default value in `providers/Onboarding` to `[['', 0]]` 

```suggestion
  const [tokenName, setTokenName] = useState(config.tokens.name)
  const [tokenSymbol, setTokenSymbol] = useState(config.tokens.symbol)
  const [commonPool, setCommonPool] = useState(config.tokens.commonPool)

  const [members, setMembers] = useState(holders)
```